client/competitionRegistration.js

```diff
@@ -111,7 +111,7 @@ Template.competitionRegistration.helpers({
 
   registrationAskAboutGuests: function() {
     var competitionId = this.competitionId;
-    competition = Competitions.findOne({ _id: competitionId }, { fields: { registrationAskAboutGuests: 1 } });
+    var competition = Competitions.findOne({ _id: competitionId }, { fields: { registrationAskAboutGuests: 1 } });
```

Comment:
Woah! It's unfortunate that our jslint didn't catch that =(. I've filed https://github.com/cubing/ccm/issues/227 to track improving this.